core/config-manager/configurationmanager_test.go

```diff
@@ -254,6 +246,34 @@ ssl:
   certPwdFile:
 
 commonkey3 : filesource
+`)
+
+	lbBytes := []byte(`
+--- 
+cse: 
+  loadbalance: 
```

Comment:
use real meaningful conf

Reply:
Meaningfull conf you mean lbBytes name i should change?
Because whatever configuration i used that is related to loadbalance i have not used any dummy data..all configuration attributes are real.

Reply:
changed lbBytes to loadbalanceConf.